client/src/components/topics.tsx

```diff
@@ -52,9 +54,17 @@ function Topics(args: {
 
   const topicButtons = topicQuestions.map((tq, i) => {
     return shouldDisplayPortrait() && displayHistoryButton ? (
-      <div data-cy={`topic-${i}`} className="slide topic-slide" key={i}>
+      <div
+        data-cy={`topic-${i}`}
+        className={
+          tq.topic === "History"
+            ? `slide topic-slide History`
```

Comment:
why do we have inconsistent style for class names? I think we probably never want capitalized?

Reply:
changed to lowercase

Reply:
I don't see that commit? A good practice if you want to say "done" to a change request, is just put the commit SHA in the comment and then you don't need to write anything else